mne/gui/_backend.py

```diff
@@ -0,0 +1,61 @@
+"""Deal with pyface backend issues"""
+# Author: Christian Brodbeck <christianbrodbeck@nyu.edu>
+#
+# License: BSD (3-clause)
+import sys
+try:
+    from pyface.api import warning
+except ImportError:
+    warning = None
+
+
+def _pyface_backend_info():
+    """Check the currently selected Pyface backend
+
+    Returns
+    -------
+    backend : str
+        Name of the backend.
+    result : 0 | 1 | 2
+        0: the backend has been tested and works.
+        1: the backend has not been tested.
+        2: the backend has known issues.
+
+    Notes
+    -----
+    See also: http://docs.enthought.com/pyface/
+    """
+    try:
+        from traits.trait_base import ETSConfig
+    except ImportError:
+        return None, 2
+
+    backend = ETSConfig.toolkit
```

Comment:
Have other people tested other platforms/backends than the two I included below?